buildbot/README.md

```diff
@@ -69,6 +69,8 @@ You can also specify job configuration options as further POST parameters:
 - `skipseashell`, which lets you supply plain HLS C code as input.
 - `estimate`, to use the Xilinx toolchain's resource estimation facility. The job will skip synthesis and execution on the FPGA.
 - `make`, to use a Makefile instead of the built-in compilation workflow (see below).
+- `hwname`, which lets you provide a name for the job during makefile flow.
+- `directives`, which let's you provide the name of a file with a set of directives (pragmas). If you want to force default directive files to be ignored you can use this as `directives=0`.
```

Comment:
Also, Buildbot (with this change) doesn’t seem to assign any particular semantics to `directives=0`; it just blindly passes it along to the Makefile (and doesn’t do anything special with it in the non-Makefile path). So maybe this description would be better left to the Makefile, which actually interprets the “zero” as something special. Or at least clarifying that this is not the Buildbot’s territory.

Reply:
I see, it doesn't indeed do anything for non-Makefile path. Why added it was to say that `DIRECTIVES=` or `DIRECTIVES= " "` wouldn't work as they'd skip the `get('directives')` and use the default file, in makefile path. I assume `0` worked because `sds++ -hls-tcl 0` also worked as intended.

Reply:
Another alternative, FWIW, is to *not* do `.get()` stuff and make the presence of an empty value (`-F directives=`) actually different from leaving it off altogether.